third_party/png.BUILD

```diff
@@ -28,6 +28,8 @@ cc_library(
         "pngwrite.c",
         "pngwtran.c",
         "pngwutil.c",
+        "powerpc/filter_vsx_intrinsics.c",
```

Comment:
Could you include these conditionally, just like here:
https://github.com/tensorflow/tensorflow/blob/master/third_party/png.BUILD#L32